packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -0,0 +1,522 @@
+                    role="region"
+                    aria-label={formatMessage(PRODUCTS_REGION_LABEL)}
+                >
+                    <VStack spacing={2}>
+                        {basket.productItems.map((item, idx) => {
+                            // Merge product details into item
+                            const productDetail = productsMap[item.productId] || {}
+                            const variant = {...item, ...productDetail}
+                            // Use findImageGroupBy to get the image
+                            const image = findImageGroupBy(productDetail.imageGroups, {
+                                viewType: 'small',
+                                selectedVariationAttributes: variant.variationValues
+                            })?.images?.[0]
+                            const imageUrl = image?.disBaseLink || image?.link || ''
+                            const addressKey = item.productId + '-' + idx
+                            const selectedAddressId =
+                                selectedAddresses[addressKey] || addresses[0]?.addressId
+
+                            return (
+                                <Box
+                                    key={addressKey}
+                                    border="1px solid"
+                                    borderColor="gray.200"
+                                    borderRadius="md"
+                                    p={4}
+                                    bg="white"
+                                    w="100%"
+                                    data-testid="multi-shipping-card"
+                                    overflow="visible"
+                                    role="article"
+                                    aria-labelledby={`product-title-${addressKey}`}
+                                    aria-describedby={`product-description-${addressKey}`}
+                                >
+                                    <Flex
+                                        direction={{base: 'column', md: 'row'}}
+                                        align="flex-start"
+                                        w="100%"
+                                        gap={{base: 4, md: 6}}
+                                    >
+                                        {/* Left: Image and info */}
+                                        <Flex direction="row" align="flex-start" flex={1} minW={0}>
+                                            <HStack align="flex-start" spacing={3} w="100%">
+                                                <Image
+                                                    src={imageUrl}
+                                                    alt={formatMessage(IMAGE_ALT, {
+                                                        productName: item.productName
+                                                    })}
+                                                    w={{base: '60px', md: '90px'}}
```

Comment:
why are there abitrary numbers? Can we use flex column instead of a set widh and height here?